vendor_blueprints/verify_refresh_status.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import sys
+import pydomo
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_FINAL_STATUS_SUCCESS = 0
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_ACCOUNT = 201
+EXIT_CODE_DATASET_NOT_FOUND = 202
+EXIT_CODE_EXECUTION_ID_NOT_FOUND = 203
+EXIT_CODE_FINAL_STATUS_INVALID = 204
+EXIT_CODE_FINAL_STATUS_CANCELLED = 205
+EXIT_CODE_STATUS_INCOMPLETE = 206
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--secret-key', dest='secret_key', required=True)
+    parser.add_argument('--dataset-id', dest='dataset_id', required=True)
+    parser.add_argument('--execution-id', dest='execution_id', required=False)
+    args = parser.parse_args()
+    return args
+
+
+def get_execution_details(dataset_id, execution_id, domo):
+    """
+    Gets the Stream ID of a particular stream using the dataSet id.
+    
+    Returns:
+        stream_id (int): the Id of the found stream 
+    """
+    streams = domo.streams
+    limit = 1000
+    offset = 0
```

Comment:
Same comment as the other PR. What happens if there are more than 1000? Don't we need looping logic here?